lib/librato/rails/subscribers/controller.rb

```diff
@@ -2,26 +2,27 @@ module Librato
   module Rails
     module Subscribers
 
-      # Controllers
+      # ActionController
 
-      ActiveSupport::Notifications.subscribe 'process_action.action_controller' do |*args|
+      ActiveSupport::Notifications.subscribe "process_action.action_controller" do |*args|
 
         event = ActiveSupport::Notifications::Event.new(*args)
         exception = event.payload[:exception]
+        tags = {
```

Comment:
Does it make sense to re-add the `rails_action` suite and only submit these tags when enabled? If so, is that name still applicable, or should it be renamed to something more explicit since it would  include `controller`, `action` and `format`?